### PR TITLE
FIT-364 Add new jenkinsfileDeploy with shared library usage

### PR DESCRIPTION
**Pull request by LotharII**

### CODE REVIEW

**Comment by LotharII** [comment]
retest this please

File: JenkinsfileDeploy [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import groovy.transform.Field
+@Library('jenkins-pipeline-utils') _
+
+@Field
+def githubCredentialsId = '433ac100-b3c2-4519-b4d6-20node7c029a103b'
+@Field
+def deAnsibleGithubUrl = 'git@github.com:ca-cwds/de-ansible.git'
+@Field
+triggerProperties = pullRequestMergedTriggerProperties('geo-services-api')
```
Comment (SergiiRedchuk) [inline JenkinsfileDeploy]: please add 'def'

File: src/test/resources/fixtures/addressSuggestResponse.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
     "state": "CA",
     "zip": "95747",
     "zip_extension": "4618",
-    "longitude": -121.3223,
-    "lattitude": 38.79895,
+    "longitude": -121.32451,
+    "lattitude": 38.79896,
     "deliverable": true
   }
 ]
```
Comment (NagaPradeepGajula-osi) [inline src/test/resources/fixtures/addressSuggestResponse.json]: new line concern

File: src/test/resources/fixtures/addressValidateResponse.json [hunk 3]
```diff
@@ -5,8 +5,8 @@
     "state": "CA",
     "zip": "95661",
     "zip_extension": "3714",
-    "longitude": -121.25118,
-    "lattitude": 38.74037,
+    "longitude": -121.25125,
+    "lattitude": 38.74059,
     "deliverable": true
   }
 ]
```
Comment (NagaPradeepGajula-osi) [inline src/test/resources/fixtures/addressValidateResponse.json]: same as above

**Pull request merged**